src/java/org/apache/cassandra/schema/SchemaPullVerbHandler.java

```diff
@@ -32,6 +32,7 @@
  * Sends it's current schema state in form of mutations in response to the remote node's request.
  * Such a request is made when one of the nodes, by means of Gossip, detects schema disagreement in the ring.
  */
+@Deprecated
```

Comment:
This will explode on rebase because of [CASSANDRA-18912](https://issues.apache.org/jira/browse/CASSANDRA-18912) We need to add `since` and maybe even a bit more info like why it was deprecated and when it is for removal (if ever)